src/item.cpp

```diff
@@ -712,6 +712,9 @@ item item::split( int qty )
 bool item::is_null() const
 {
     // Actually, type should never by null at all.
+    if( !this ) {
```

Comment:
```suggestion
    if( this == nullptr ) {
```
Not exactly sure, but it's my best guess

Reply:
That is not allowed either. As BrettDong mentioned above, it is undefined behavior when the item is null. 

It looks to me that the item::is_null() method is not repairable - and it gets called from a lot of places.

I think I'll try filtering how the items are presented for insertion instead, so that the ammo is not shown.